joint_trajectory_controller/test/test_trajectory_actions.cpp

```diff
@@ -1012,3 +1021,74 @@ INSTANTIATE_TEST_SUITE_P(
     std::make_tuple(
       std::vector<std::string>({"effort"}),
       std::vector<std::string>({"position", "velocity", "acceleration"}))));
+
+TEST_P(TestTrajectoryActionsTestScalingFactor, test_scaling_execution_time_succeeds)
```

Comment:
I have the feeling that just adding one test for such a complex feature is probably not enough. 

I think it is missing some parts:

1. It just checks the if the trajectory took the right amount of time -> It does not check if the trajectory was executed in the right way (e.g. had a spike what ever)

2. Executing multiple trajectories in series 
3. Proper behavior regarding state and goal tolerances (and goal time tolerances)

2/3. are motivated by: https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/882